.gitignore

```diff
@@ -0,0 +1,4 @@
+importers/__pycache__
+requirements.txt
```

Comment:
@ThanGrove the requirements file needs to be included in this repo. Was this addition intentional?